ui/sortable.js

```diff
@@ -830,6 +828,16 @@ return $.widget("ui.sortable", $.ui.mouse, {
 
 	},
 
+	_createTrPlaceholder: function( sourceTr, targetTr ) {
+		var that = this;
+
+		sourceTr.children().each(function() {
+			$( "<td>&#160;</td>", that.document[0] )
```

Comment:
I know you just copy/pasted this, but since other code needs to be updated, might as well fix the spacing here as well.